examples/pytorch/language-modeling/run_clm.py

```diff
@@ -391,7 +392,9 @@ def main():
             cache_dir=model_args.cache_dir,
             revision=model_args.model_revision,
             use_auth_token=True if model_args.use_auth_token else None,
-            torch_dtype=model_args.torch_dtype,
+            torch_dtype=model_args.torch_dtype
+            if model_args.torch_dtype in ["auto", None]
+            else getattr(torch, model_args.torch_dtype),
```

Comment:
Let's put this on its own line before this model call for readability.

Reply:
done